### ISSUE TITLE
.eleventyignore: includes directory not ignored

### ISSUE THREAD

**Issue by kleinfreund** [11ty/eleventy#403]
I’ve set my *input directory* to `"src"` and my *includes directory* to `""`; thus, they’re the same.

An unfortunate by-product of this is that files in the includes directory are processed. My layouts end up as transformed HTML in my output directory. Therefore, I tried adding an `.eleventyignore` file to explicitly exclude it. This doesn’t seem to work.

.eleventyignore:
```
src/_includes/**
```

Some related debugger output:

```
  Eleventy:EleventyFiles /home/phil/dev/repos/kleinfreund.de/.eleventyignore,src/.eleventyignore ignoring: !/home/phil/dev/repos/kleinfreund.de/src/_includes/**/* +0ms
  Eleventy Directories:
  Eleventy Input: src
  Eleventy Data: src/_data
  Eleventy Includes: src
  Eleventy Output: _site
  Eleventy Template Formats: md,liquid,html +0ms

  ...
  Eleventy:EleventyFiles Searching for: [
    './src/**/*.md',
    './src/**/*.liquid',
    './src/**/*.html',
    '!/home/phil/dev/repos/kleinfreund.de/node_modules/**',
    '!/home/phil/dev/repos/kleinfreund.de/_site/**',
    '!/home/phil/dev/repos/kleinfreund.de/src/_includes/**',
    '!./_site/**',
    '!./src/_data/**'
  ] +51ms
```

Is this to be expected?

**Comment by chrisdmacrae** [comment]
To set your includes directory to the same as your src directory, do `'./'` not `""‘ 

And your eleventy ignore wouldn't work because your includes are not in `src/_includes/‘ as I understand it?

**Comment by kleinfreund** [comment]
> To set your includes directory to the same as your src directory, do `'./'` not `""‘

This might have the same effect, but setting includes to `""` is sufficient.



> And your eleventy ignore wouldn't work because your includes are not in `src/_includes/‘ as I understand it?

My includes are in `src/_includes`. The contents of the file are explicitly ignored properly; however, they are processed anyway due to the `'./src/**/*.liquid'` entry appearing before the `'!/home/phil/dev/repos/kleinfreund.de/node_modules/**'` in the output I posted above.

**Comment by chrisdmacrae** [comment]
My bad, I misunderstood. 

11ty uses fast-glob, so the order of the globs does not matter. Ignores are always evaluated first. 

So given that, I'm not sure why that wouldn't work. 

Maybe try `src/_includes/**/*.liquid`?

**Comment by zachleat** [comment]
Ah, looks like an issue upstream in our dependency. I think if we convert those eleventyignore paths to relative instead of absolute paths it seems to work okay. Code coming soon

**Comment by zachleat** [comment]
This will be included with 0.7.2.
